raiden/tests/integration/fixtures/blockchain.py

```diff
@@ -193,6 +202,31 @@ def blockchain_backend(
     raise ValueError('unknow cluster type {}'.format(blockchain_type))
 
 
+def deploy_contract_web3(
+    contract_name: str,
+    poll_timeout,
+    deploy_client,
+    *args
+):
+    web3 = deploy_client.web3
+
+    contract_interface = CONTRACT_MANAGER.abi['SecretRegistry']
```

Comment:
Please don't use the contract names directly - use the imported names from the `raiden-contracts` package.